### DISCUSSION TITLE
Downloading a file from URL

### DISCUSSION THREAD

**Discussion by tautologias345** [godotengine/godot-proposals#5695] [Scripting]
For example, a bundle with many games, to that the bundle don't have much filesize such as BlueMaxima's Flashpoint Ultimate, the user with internet can download games to have on `res://` such as BlueMaxima's Flashpoint Infinity. Once downloaded, the user can play offline.

**Comment by Mikeysax** [comment]
Did you look at this? https://docs.godotengine.org/en/stable/tutorials/networking/http_request_class.html
